app/scenes/options_modal/options_modal_list.android.js

```diff
@@ -20,13 +22,25 @@ export default class OptionsModalList extends PureComponent {
         const {items, onCancelPress} = this.props;
 
         const options = items.map((item, index) => {
+            let textComponent;
+            if (typeof item.text === 'string') {
```

Comment:
Maybe flip this around and make it an if-else (without the else if) so that it checks for a localized value and if not just show the contents of text? That way it'll still print something if someone passes in something else so whoever's using it can see they're using it wrong